pkg/chartverifier/certifier.go

```diff
@@ -67,12 +76,9 @@ func (c *certifier) Certify(uri string) (*Certificate, error) {
 	procExec := exec.NewProcessExecutor(c.settings.Debug)
 	oc := tool.NewOc(procExec)
 
-	osVersion, err := oc.GetVersion()
+	osVersion, err := oc.GetVersion(c.openshiftVersion)
 	if err != nil {
-		// NOTE(isutton): oc or a suitable kubeconfig might not be
-		//                available, so for now we leave it as "N/A"
-		//                instead of raising an error.
-		osVersion = "N/A"
+		return nil, OpenShiftVersionErr{err.Error()}
```

Comment:
Revert `oc.GetVersion()` and in case of error use `c.openshiftVersion` as fallback, and then erroring if neither is available.

Reply:
Reverted. 03bd100976c1685bdec2a689611f61d3e2371157